grpc.go

```diff
@@ -193,21 +193,25 @@ func updateContainerCpuset(cgroupPath string, newCpuset string, cookies cookie)
 }
 
 func (a *agentGRPC) onlineCPUMem(req *pb.OnlineCPUMemRequest) error {
-	if req.NbCpus <= 0 {
+	if req.NbCpus == 0 && req.CpuOnly {
```

Comment:
What about `NbCpus < 0`? We should fail in such case.

Reply:
`NbCpus` is uint32. I think there is no need to consider `NbCpus <0`.

Reply:
heh, good point! I thought it was int32 as we had `<=` there ;)